src/Lib.hs

```diff
@@ -294,8 +317,11 @@ filterAnalyzedTickets ticketsInfo =
     isTicketAnalyzed TicketInfo{..} = (renderTicketStatus AnalyzedByScriptV1_0) `notElem` (getTicketTags tiTags)
     -- ^ This is showing that something is wrong...
 
+    unsolvedTicketStatus :: [TicketStatus]
+    unsolvedTicketStatus = TicketStatus <$> ["new", "open", "hold", "pending"]
```

Comment:
Use `map` when lifting in a list context, makes it more explicit and easier to read...